community/louvain_common.go

```diff
@@ -35,11 +35,80 @@ func Q(g graph.Graph, communities [][]graph.Node, resolution float64) float64 {
 	}
 }
 
+// Multiplex is a multiplex graph.
```

Comment:
Could you please define multiplex graph for me, what are its main characteristics? Is this https://en.wikipedia.org/wiki/Multidimensional_network a relevant reference?